vuepress/docs/next/docs/component-generator/blueprint-features.md

```diff
@@ -0,0 +1,130 @@
+# Entando Blueprint Features
+
+The Entando Blueprint makes it easier and faster to customize your application by generating controllers, repositories, services, and micro frontends for your entity. It means generating those project files in minutes, simply by answering questions that define the parameters of your application. 
+
+The Entando Blueprint uses JHipster technology based on Embedded JavaScript (EJS), a templating language that provides powerful constructs for large scale file generation. 
+
+## The Features:
+* Backend with Spring Boot 
+  * Profiles (dev, prod)
+  * Preconfigured Cross-Origin Resource Sharing (CORS) settings
+  * Keycloak integration 
+  * Swagger/OpenAPI frontend
+  * Data modeling 
+  * Jhipster Design Language (JDL) support for schema upgrade
```

Comment:
JHipster (capital H)

Reply:
This bullet overlaps with the next one. Let's keep this as just "JHipster Design Language (JDL) support" and the  next one as-is. The JDL support is useful even without Liquibase.